sdk/spring/spring-cloud-azure-autoconfigure/src/test/java/com/azure/spring/cloud/autoconfigure/user/agent/amqp/servicebus/ServiceBusProcessorUserAgentTest.java

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.cloud.autoconfigure.user.agent.amqp.servicebus;
```

Comment:
how about using package `telemetry` instead of `user.agent`?

Reply:
No. `telemetry` has wider scope. `user-agent` is more specific and straightforward.

Reply:
Then please change user.agent to useragent, the package user doesn't make any sense.

Reply:
Updated.